src/applications/disability-benefits/all-claims/pages/paymentInformation.js

```diff
@@ -9,45 +11,64 @@ const {
 } = fullSchema.properties;
 
 export const uiSchema = {
-  'ui:title': 'Payment information',
-  'ui:description':
-    'This is the bank account information we have on file for you. We’ll pay your disability benefit to this account.',
-  bankAccountType: {
-    'ui:title': 'Account type',
+  'view:bankAccount': {
+    'ui:title': 'Payment Information',
+    'ui:description':
+      'We’re currently paying your compensation to this account',
+    'ui:field': ReviewCardField,
     'ui:options': {
-      widgetClassNames: 'va-select-medium-large',
+      viewComponent: PaymentView,
+      reviewTitle: 'Payment information',
+      editTitle: 'Add new bank account',
+      itemName: 'account',
+      startInEdit: formData =>
+        Object.keys(formData).every(key => !formData[key]),
+      volatileData: true,
     },
-    'ui:required': bankFieldsHaveInput,
-  },
-  bankAccountNumber: {
-    'ui:title': 'Account number',
-    'ui:options': {
-      widgetClassNames: 'va-input-medium-large',
+    bankAccountType: {
+      'ui:title': 'Account type',
+      'ui:options': {
+        widgetClassNames: 'va-select-medium-large',
+      },
+      'ui:required': bankFieldsHaveInput,
```

Comment:
To make this work properly, `bankFieldsHaveInput` will need to be updated with the new, nested path for the bank info (`formData[view:bankAccount]`)

Reply:
Oh, hey. This must be why the above happened. :bulb:

Good catch.